src/sentry/search/events/fields.py

```diff
@@ -2365,15 +2365,13 @@ def resolve_orderby(self, orderby: Optional[Union[List[str], str]]) -> List[Orde
                 if isinstance(selected_column, Column) and selected_column == resolved_orderby:
                     validated.append(OrderBy(selected_column, direction))
                     continue
-                elif (
-                    isinstance(selected_column, Function) and selected_column.alias == bare_orderby
-                ):
-                    validated.append(OrderBy(selected_column, direction))
-                    continue
 
-            # TODO: orderby aggregations
+                if is_function(bare_orderby):
+                    bare_orderby = get_function_alias(bare_orderby)
```

Comment:
could we do this after L2360 so we only call `get_function_alias` once per `orderby_column`